src/util/system.cpp

```diff
@@ -1148,6 +1153,38 @@ void runCommand(const std::string& strCommand)
 }
 #endif
 
+#ifdef USE_EXTERNAL_SIGNER
+UniValue runCommandParseJSON(const std::string& strCommand, const std::string& strStdIn)
```

Comment:
In commit "[util] add runCommandParseJSON" (82079a04e6505424bbcb4b030a81d368933d24a7)

Naming convention for argument and variables in this function could use snake_case instead of camelCase and drop strHungarianNotationPrefixes